### ISSUE TITLE
Support Checkstyle 6.10

### ISSUE THREAD

**Issue by mmoayyed** [jshiell/checkstyle-idea#184]
Currently:

```
cannot initialize module TreeWalker - Unable to instantiate CommentsIndentation The file has been blacklisted for 60s.
```

**Comment by jshiell** [comment]
Until this is updated you can use an older version of the Google checks than doesn't rely on CommentsIndentation. I'll try get it sorted this week, mind.

**Comment by jshiell** [comment]
Released as 4.19.
